qiskit/aqua/operators/list_ops/summed_op.py

```diff
@@ -57,18 +58,57 @@ def distributive(self) -> bool:
         return True
 
     def add(self, other: OperatorBase) -> OperatorBase:
+        """ Return Operator addition of self and other, overloaded by ``+``.
+            Note that this appends other to ``self.oplist`` without checking other is already
+            included or not. If you want to simplify this, please use ``SummedOp.simplify``.
```

Comment:
```suggestion
        """Return Operator addition of self and other, overloaded by ``+``.
            
        Note that this appends other to ``self.oplist`` without checking other is already
        included or not. If you want to simplify this, please use ``SummedOp.simplify``.
```

Reply:
Thank you for your comments. I will use `Note`.